assets/agenda/components/AgendaListItemIcons.jsx

```diff
@@ -87,6 +105,21 @@ class AgendaListItemIcons extends React.Component {
                     </div>
                 )}
 
+                <div>
+                    {subject.length != 0 && (
+                        <span className="label label--rounded">
```

Comment:
I would expect each one to be a separate label, not just 1 label for all. and to have that `subject--` class on the same level so it can be used as a modifier for that

Reply:
these are the outer label class which Fritz suggested. and we can modify the label here 
https://github.com/superdesk/newsroom-core/pull/420/files/7347c50aba955604de9bffeec2bfee8ccfc3d44d#diff-4799aff35d2a8cf74c640e3308b36e2b2eadec8a6a4389f3868bd7edf029044fR114